### ISSUE TITLE
SSR not working

### ISSUE THREAD

**Issue by cazala** [decentraland/ui#38]
Currently this lib doesn't work on Next.js out of the box because of the server-side rendering. This is because we make use of `window` and that breaks on the server, things like this:

```
componentWillMount() {
  window.addEventListener(...)
}
```

To fix this we can simply check first if `window` exists:

```
componentWillMount() {
  if (window) {
    window.addEventListener(...)
  }
}
```

**Comment by dev3288** [comment]
any solution for this ?

**Comment by cazala** [comment]
Closed by #60
